README.rst

```diff
@@ -175,9 +175,11 @@ kickstart file, the target of a substitution is any string starting with
 
 Configuration parameters may also come from special shell scripts that are
 sourced during run_kickstart_tests.sh.  It will first look at the defaults in
-scripts/defaults.sh.  It will then look at any user-specific defaults in
-~/.kstests.defaults.sh.  These take precedence over the local environment.
-Environment variables set on the command line have the highest priority.
+scripts/defaults.sh.  Next, if platform is specified using -p PLATFORM option,
+the scripts/defaults-PLATFORM.sh file is sourced.  Finally it will source any
+user-specific defaults in ~/.kstests.defaults.sh.  These take precedence over
+the local environment.  Environment variables set on the command line have the
+highest priority.
```

Comment:
After discussing with Radek, this sentence needs calling out. It is true neither before this PR nor after it's merged. It is part of the diff only because it's pushed in front of the changes and re-wrapped at line length. However PR #682 does make the sentence true.